### ISSUE TITLE
Closing animation is laggy in certain scenarios

### ISSUE THREAD

**Issue by MoeAmen** [brave/brave-browser#42753]
### Description

Animation of tab closing and opening is laggy in certain scenarios as shown in the recordings

### Steps to reproduce

1. Open Brave
2. Close and open tabs
3. Usually is shown when a website is visited and not in new tab

### Actual result

Animation is laggyhttps://github.com/user-attachments/assets/a98ee1c0-71f6-4c3b-b254-1bb6593bc4b7

### Expected result

Animation should be smoothhttps://github.com/user-attachments/assets/a0f15e05-9f14-46e7-8a3b-572e83ced684

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.73.97

### Channel information

- [x] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [x] with Brave Shields disabled
- [x] with Brave Rewards disabled
- [x] in the latest version of Chrome

### Miscellaneous information

Earlier issue was opened, but I did not include recordings in that one, please close the older one

**Comment by bugproof** [comment]
Yes, this is very apparent when compared to other browsers. To the point I was thinking browser runs in 60hz refresh rate

**Comment by bugproof** [comment]
https://github.com/brave/brave-core/blob/861ebc58753e833326d4298f4bfe57e50c6fafda/browser/ui/views/tabs/brave_tab_container.cc#L233-L262

this is more like a chromium issue, the same "laggy" effect is present in regular Google Chrome
